Library/include/CSP/Systems/Assets/AssetCollection.h

```diff
@@ -71,16 +71,37 @@ class CSP_API AssetCollection
 	/// @return AssetCollection : immutable reference
 	const csp::common::Map<csp::common::String, csp::common::String>& GetMetadataImmutable() const;
 
+	// The unique identifier of this asset collection.
 	csp::common::String Id;
+
+	// The unique name of this asset collection.
 	csp::common::String Name;
+
+	// The type of this asset collection.
 	EAssetCollectionType Type;
+
+	// The set of tag strings that have been associated with this asset collection. Note that asset collections can be searched by tag.
 	csp::common::Array<csp::common::String> Tags;
+
+	// The unique identifier of the POI this asset collection relates to. Empty if it does not relate to a POI.
 	csp::common::String PointOfInterestId;
+
+	// The unique ID of the asset collection that is a parent to this asset collection. Empty if there is no parent relationship.
 	csp::common::String ParentId;
-	csp::common::Array<csp::common::String> SpaceIds;
+
+	// Where the asset collection belongs to a space, the unique ID of the space that this asset collection belongs to. Empty otherwise.
+	csp::common::String SpaceId;
```

Comment:
What's the rationale behind this change? This essentially limits us in terms of sharing assets between spaces, and requires a unique copy on CHS for each space. I would rather we instead give the user the _option_ to copy the assets instead of referencing the original.

Reply:
This was discussed in yesterday's engineering meeting with all client teams, CSP and CHS present. We agreed on this approach unanimously.

Reply:
Understood, however since the group consensus was to adopt this approach, and our services team are also considering doing similarly at their API level, I don't see any harm in including this part in the PR. It helps keeps the new `Copy` function cleaner too!